src/Sylius/Bundle/FlowBundle/spec/Process/Builder/ProcessBuilderSpec.php

```diff
@@ -31,4 +34,27 @@ function it_is_process_builder()
     {
         $this->shouldImplement(ProcessBuilderInterface::class);
     }
+
+    function it_should_not_add_inactive_steps(
+        StepInterface $step,
+        ProcessScenarioInterface $scenario
+    )
+    {
+        $this->build($scenario);
+        $step->isActive()->willReturn(false);
+        $step->setName(Argument::any())->shouldNotBeCalled();
+        $this->add('foobar', $step);
```

Comment:
I would add blank line before, to improve readability (as well as in next spec).